its/projects/MultiLanguageSupport/frontend/PageOne.js

```diff
@@ -1,22 +1 @@
-import React from 'react';
-
-class PageOne extends React.Component {
-    constructor(props) {
-        var message = 'Welcome to PageOne!'; // javascript:S3504
-        super(props);
-        this.state = {
-            message: message,
-        };
-    }
-
-    render() {
-        return (
-            <div>
-                <h1>Page One</h1>
-                <p>{this.state.message}</p>
-            </div>
-        );
-    }
-}
-
-export default PageOne;
\ No newline at end of file
+if (a & b) { /* ... */ } // javascript:S1529 - The operator & is used in a boolean context
```

Comment:
For some reason the other rule did not raise, but this one does.

Reply:
Yes, it is the quality profile.
We are pinning the [javascript plugin](https://github.com/SonarSource/sonar-scanner-msbuild/blob/master/its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/Tests.java#L42) version, so we are a bit behind on the rules.